pyproject.toml

```diff
@@ -1,18 +1,14 @@
 [tool.poetry]
 name = "nekontrol"
 version = "0.2.5"
-description = "A program for testing kattis solutions with input and output."
+description = "A program for testing Kattis solutions with input and output."
 authors = [
     "Emanuel S <emanuel@empa.xyz>",
 ]
 readme = "README.md"
 homepage = "https://github.com/Quaqqer/nekontrol"
 repository = "https://github.com/Quaqqer/nekontrol"
 keywords = ["kattis", "competitive programming"]
-packages = [
-    { include = "nekontrol", from = "src" },
-]
-include = ["CHANGELOG.md"]
```

Comment:
Should packages really be removed?

Reply:
Alright, I can see what you mean. To me it is more nice to declare the project structure than having it be implicit.